dnf-docker-test/features/module-updateinfo-1.feature

```diff
@@ -0,0 +1,144 @@
+        """
+        ---
+        document: modulemd
+        version: 2
+        data:
+          name: test-module
+          stream: 9
+          version: 1
+          context: 6c81f848
+          arch: noarch
+          summary: Test module
+          description: >-
+            Test is an advanced Object-Relational database management system (DBMS).
+          license:
+            module:
+            - MIT
+            content:
+            - MIT
+          profiles:
+            client:
+              rpms:
+              - Test
+          artifacts:
+            rpms:
+            - Test-0:9.6.8-1.module.noarch
+        ...
+        ---
+        document: modulemd
+        version: 2
+        data:
+          name: test-module
+          stream: 10
+          version: 1
+          context: 6c81f848
+          arch: noarch
+          summary: PostgreSQL module
+          description: >-
+            Test is an advanced Object-Relational database management system (DBMS).
+          license:
+            module:
+            - MIT
+            content:
+            - MIT
+          profiles:
+            client:
+              rpms:
+              - Test
+          artifacts:
+            rpms:
+            - Test-0:10.6-1.module.noarch
+        ...
+        ---
+        document: modulemd-defaults
+        version: 1
+        data:
+            module: test-module
+            stream: 9
+            profiles:
+              9: [client]
+              10: [client]
+        ...
+        """
+    And a file "modules.yaml" with type "modules" added into repository "updates"
+        """
+        ---
+        document: modulemd
+        version: 2
+        data:
+          name: test-module
+          stream: 10
+          version: 2
+          context: 6c81f848
+          arch: noarch
+          summary: PostgreSQL module
+          description: >-
+            Test is an advanced Object-Relational database management system (DBMS).
+          license:
+            module:
+            - MIT
+            content:
+            - MIT
+          profiles:
+            client:
+              rpms:
+              - Test
+          artifacts:
+            rpms:
+            - Test-0:10.7-1.module.noarch
+        ...
+        """
+
+
+@bz1622614
+Scenario: List available updates for installed streams (updates available)
+  When I enable repository "base"
+   And I successfully run "dnf -y module install test-module:10"
+  When I enable repository "updates"
+   And I successfully run "dnf updateinfo --list"
+  Then the command stdout should match regexp "RHEA-2018:101 enhancement Test-10.7-1.module.noarch"
+
+
+Scenario: Cleanup from previous scenario
+  When I successfully run "dnf -y module remove test-module:10"
+   And I successfully run "dnf -y module reset test-module"
+   And I disable repository "updates"
+
+
+# this scenario is failing on upstream, but should pass on lates RHEL8
+@xfail
```

Comment:
why @xfail? it seems to work on RHEL-8.0. Can you remove it?

Reply:
It is failing on upstream dnf, because it requests changes not yet merged into upstream. I created new pull request on rhel-8.0.0 branch without xfail tag.